python/.flake8

```diff
@@ -9,10 +9,9 @@ ignore =
     E203
 
 [pydocstyle]
-match = ^(.*abc\.py|.*api/types\.py|.*single_column_frame\.py|.*indexed_frame\.py)$
-# Due to https://github.com/PyCQA/pydocstyle/issues/363, we must exclude rather than include using match-dir.
-match-dir = ^(?!ci|cpp|python/dask_cudf|python/cudf_kafka|python/custreamz).*$
-# In addition to numpy style, we additionally ignore:
+match = ^(.*abc\.py|.*api/types\.py|.*single_column_frame\.py|.*indexed_frame\.py|.*series\.py)$
+# Due to https://github.com/PyCQA/pydocstyle/issues/363, we must exclude rather than include using match-dir. Note that as discussed in https://stackoverflow.com/questions/65478393/how-to-filter-directories-using-the-match-dir-flag-for-pydocstyle, unlike the match option above this match-dir will have no effect when pydocstyle is invoked from pre-commit. Therefore this exclusion list must also be maintained in the pre-commit config file.
+match-dir = "^(?!(ci|cpp|python/dask_cudf|python/cudf_kafka|python/custreamz|python/cudf)).*$"
```

Comment:
It surprises me that these are relative to the repo root rather than relative to `python/.flake8`.

Reply:
This is a good point. I copy-pasted at some point from the changes in `.pre-commit-config.yaml`, which lives at the root. It's probable that in doing so I overwrote the correct paths in .flake8. I'll check this and comment again once I know for sure.